vars/handlePipelineStepErrors.groovy

```diff
@@ -59,14 +64,19 @@ void call(Map parameters = [:], body) {
     try {
         if (config.echoDetails)
             echo "--- Begin library step of: ${config.stepName} ---"
-
-        body()
-    } catch (AbortException ae) {
+        if (!config.failOnError && config.stepTimeouts && config.stepTimeouts[config.stepName]) {
+            timeout(time: config.stepTimeouts[config.stepName]) {
+                body()
+            }
+        } else {
+            body()
+        }
+    } catch (AbortException | org.jenkinsci.plugins.workflow.steps.FlowInterruptedException err) {
```

Comment:
1.) [clear naming] should not be `err`, but `ex` since we do not deal with `Error`s here but with `Exceptions`.
2.) what about using an import statement for the `FlowInterruptedException`?

Reply:
good points,  will change that ...